enterprise/server/remote_execution/execution_server/execution_server.go

```diff
@@ -633,14 +634,17 @@ func (s *ExecutionServer) PublishOperation(stream repb.Execution_PublishOperatio
 	}
 }
 
-func (s *ExecutionServer) updateRouter(ctx context.Context, taskID string, executeResponse *repb.ExecuteResponse) error {
```

Comment:
nit: maybe these two things (updateRouter and updateTaskAccounting) don't really belong together. Why not just make another for task accounting ?